src/components/Flexy/Flexy.Block.tsx

```diff
@@ -24,6 +24,4 @@ class Block extends React.PureComponent<Props> {
   }
 }
 
-namespaceComponent(COMPONENT_KEY.Block)(Block)
```

Comment:
🙌

Reply:
don't celebrate too quickly :) 

`namespaceComponent` is added back by propConnect, another reason I would love to get rid of this system

Reply:
Haha, well at least we'll see this function less often, so I'd still consider it a win :P